api/client/set.go

```diff
@@ -0,0 +1,75 @@
+package client
+
+import (
+	"fmt"
+
+	Cli "github.com/docker/docker/cli"
+	flag "github.com/docker/docker/pkg/mflag"
+	"github.com/docker/docker/pkg/units"
+	"github.com/docker/docker/runconfig"
+)
+
+func (cli *DockerCli) CmdSet(args ...string) error {
+	cmd := Cli.Subcmd("set", []string{"CONTAINER [CONTAINER...]"}, "Update properties of one or more containers", true)
```

Comment:
Is there any way to reuse runconfig.Parse() so we don't have to define these flags in two places?